CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ Here are a few examples:
 * New cop `ElseLayout` checks for odd arrangement of code in the `else` branch of a conditional expression. ([@bbatsov][])
 ```
 
-* Mark it up in [Markdown syntax][6].
+* Mark it up in [Markdown syntax][6] (in particular, ensure cop names are surrounded in backticks).
```

Comment:
We use backticks for a lot of things, so I'm not sure if mentioning this one use-case explicitly is a good idea.